submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="./popup-icons/favicon.png" type="image/x-icon">
+    <title>popup</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul class="nav">
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Mail</a></li>
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Image</a></li>
+                <li class="nav_item">
+                    <a href="#" target="_blank">
+                        <img class="nav-img" src="./popup-icons/bell.png" alt="bell">
+                    </a>
+                </li>
+                <li class="popup_button nav_item">
+
+                    <input class="popup_button-input" type="checkbox" name="menu" id="menu">
+                    <label class="popup_button-label" for="menu">
```

Comment:
Add `cursor: pointer` to all interactive elements. Also, fix styles on hover and focus: avoid image shifting